packages/dev/docs/src/toc.css

```diff
@@ -13,11 +13,15 @@
 .toc {
   position: fixed;
   top: 80px;
-  right: 0;
+  right: 8px;
   bottom: 0;
   overflow-y: auto;
 }
 
+.tocContainer {
+  border-left: 1px solid var(--spectrum-global-color-gray-200);
```

Comment:
ahhhh.... noooo....please make some space between the border and the content at the very least :eye-bleeding:
;)

Reply:
hahahah sorry -- forgot to push the rest. check it out now!